modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -235,5 +235,188 @@
     "org.wso2.carbon.identity.entitlement.pip.DefaultResourceFinder"
   ],
   "identity.entitlement.entitlement_engine_caching_interval": "100000",
-  "identity.entitlement.JSON_shorten_form_enabled": false
+  "identity.entitlement.JSON_shorten_form_enabled": false,
+
+
+  "identity.events.threadPool_size": "10",
```

Comment:
shall we have this as "thread_pool_size"

Reply:
Fixed by [073dd55](https://github.com/wso2/product-is/pull/4522/commits/073dd558df49c3ce182e4ec6aa1b07367c22f0ec)